test/features/comments.feature

```diff
@@ -1,24 +1,25 @@
 @comments
 Feature: Comments
 
+    @unauthenticated
     Scenario: The Add Comment form should not display for a non-logged-in user - instead they see a 'Login to comment' button
```

Comment:
unsure why we need 'commenting' in harvester. 

yes, its good to know that it does not clash, but this plugin is about harvester/geoscience import etc

Reply:
If we don't include regression testing here, then it's possible to review twice, cut a release, and open a pull request in infrastructure, before realising that there's an incompatibility that could have been detected immediately with our existing automated scenario tests.